chainio/clients/wallet/fireblocks_wallet.go

```diff
@@ -155,6 +156,23 @@ func (t *fireblocksWallet) SendTransaction(ctx context.Context, tx *types.Transa
 		}
 	}
 
+	gasLimit := ""
+	if tx.Gas() > 0 {
+		gasLimit = strconv.FormatUint(tx.Gas(), 10)
+	}
+
+	// if the gas fees are specified in the transaction, use them.
+	// Otherwise, use the default "HIGH" gas price estimated by Fireblocks
+	maxFee := ""
+	priorityFee := ""
+	feeLevel := fireblocks.FeeLevel("")
+	if tx.GasFeeCap().Cmp(big.NewInt(0)) > 0 && tx.GasTipCap().Cmp(big.NewInt(0)) > 0 {
+		maxFee = tx.GasFeeCap().String()
+		priorityFee = tx.GasTipCap().String()
```

Comment:
Do we also need to support legacy txs for whatever reason?
See in the eip
<img width="662" alt="image" src="https://github.com/Layr-Labs/eigensdk-go/assets/9342524/0e996f7c-72e7-4afc-a865-ba87a988d5ab">

Reply:
Sure, callers would usually construct an 1559 transaction. But I added `gasPrice` field in case the caller wants to construct a non-1559 transaction.